wit/deps/system/disk.wit

```diff
@@ -25,5 +25,8 @@ interface disk {
         time-flushing: option<u64>,
     }
 
-    stat: func() -> result<list<disk-stat>, string>;
+    // interval-ms is a hint equal to user loop interval,
+    // The underlying implementation will guarantee stat data
+    // is newer than interval-ms/10
```

Comment:
For most system statistics, sampling interval less than 1ms isn't of much use. Thus the minimal `interval-ms` is 1ms. 

`stat` function accepts `interval-ms` as a hint to guarantee collected statistics data is new enough and with low overhead.

Reply:
I prefer not to descript the underlying implementation details in the doc as we may slightly change it in the future.

Reply:
fixed